provider/runbook_resource_test.go

```diff
@@ -0,0 +1,325 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/firehydrant/terraform-provider-firehydrant/firehydrant"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccRunbookResource_basic(t *testing.T) {
```

Comment:
alright so I think I see what you were doing the other file was for runbook model properties and this file is for the resources nested within runbooks?

Reply:
So the other test was for the runbook data source, which is essentially a way to read information about runbooks created outside of Terraform ([you can read more about data sources here](https://www.terraform.io/language/data-sources)).

This test file is for the runbooks resource, which is used to create/update/delete runbooks.

Reply:
ah ha, okay now this makes sense. Thank you for the link and the explanation!